src/main.c

```diff
@@ -662,10 +659,7 @@ static void core_secondly()
     if (((int) (nowtm.tm_min / 5) * 5) == (nowtm.tm_min)) {     /* 5 min */
       call_hook(HOOK_5MINUTELY);
       check_botnet_pings();
-      if (!quick_logs) {
-        flushlogs();
-        check_logsize();
-      }
+
```

Comment:
why is check_logsize not necessary here but left in core_minutely()?

Reply:
`check_logsize()` is no rocket science, so its no called for every log to make it absolutely precise, instead it is called minutely.

and thats enough imo.

if it is already called minutely, i see no reason, to additionally call it every 5 minutes, making it kinda called twice every 5 minutes.

could there be some log bloat hooked for the 5 minute hook? yes, but it will then happen anyway. and it will be fixed in next minute anyway.

the difference in calling or not calling it additionally every 5 minutes is so small, that i took the freedom to remove it here.